py7zr/callbacks.py

```diff
@@ -35,6 +35,11 @@ def report_start(self, processing_file_path, processing_bytes):
         """report a start event of specified archive file and its input bytes."""
         pass  # noqa
 
+    @abstractmethod
+    def report_update(self, decompressed_bytes):
```

Comment:
Could you update the documentation to specify if this is the total number of bytes decompressed so far, or the total number of bytes decompressed since the last update.

Reply:
Yes, it could be not obvious. It is a number of bytes decompressed since the last update.